### ISSUE TITLE
Loss Not Updating During Fine-Tuning with Token-Wise Dynamic Activation Configuration

### ISSUE THREAD

**Issue by Dianaia** [ChenMnZ/PrefixQuant#14]
I'm attempting to fine-tune a model using the "fine-tuning + token-wise dynamic" configuration, aiming to achieve results similar to those in Table 13 of the paper. However, during training, I noticed that the loss per epoch does not change, and remains constant across multiple epochs, as shown below:
```
[2024-11-26 17:55:02 root](block_ap.py 299): INFO blocks 0 epoch 0 train_loss:3.6545388866215944e-06 val_loss:3.598202511057025e-06  norm:nan max memory_allocated 5569.38671875 time 11.438298225402832 
[2024-11-26 17:55:13 root](block_ap.py 299): INFO blocks 0 epoch 1 train_loss:3.6545388866215944e-06 val_loss:3.598202511057025e-06  norm:nan max memory_allocated 5569.38671875 time 11.117196559906006 
[2024-11-26 17:55:24 root](block_ap.py 299): INFO blocks 0 epoch 2 train_loss:3.6545388866215944e-06 val_loss:3.598202511057025e-06  norm:nan max memory_allocated 5569.38671875 time 11.162273406982422 
```
Could anyone suggest possible reasons for this issue or provide guidance on how to resolve it?
Here is the configuration I am using for the experiment:
```
CUDA_VISIBLE_DEVICES=0 python main.py \
--model_path path/to/llama-2-7B  \
--model_name Llama-2-7b \
--output_dir ./log/llama-2-7b-w4a8kv4 \
--wbits 4 \
--input_bits 8 \
--v_bits 4 \
--k_bits 4 \
--kv_group_size 128 \
--kv_mode static \
--mse_init \
--pre_rotate \
--qk_online_had \
--set_prefixed_tokens \
--eval_ppl \
--epochs 10
```
Any help or advice would be greatly appreciated. Thank you!

**Comment by ChenMnZ** [comment]
Please try add `--use_fp32` in your command.

**Comment by Dianaia** [comment]
> 请尝试添加`--use_fp32`您的命令。

It does work, thanks!
